packages/itwinui-css/src/side-navigation/side-navigation.scss

```diff
@@ -119,7 +119,29 @@ $iui-side-navigation-icon-margins: calc(1.5 * var(--iui-size-m));
     }
 
     &.iui-submenu-open {
-      background-color: var(--iui-color-background-hover);
+      --_iui-button-active-stripe-color: var(--iui-color-background);
+
+      &:hover {
+        --_iui-button-active-stripe-color: var(--iui-color-background-hover);
+      }
+
+      &::after {
+        content: '';
+        position: absolute;
+        inset: var(--_iui-button-active-stripe-inset);
+        background-color: var(--_iui-button-active-stripe-color);
+        transition: background var(--iui-duration-1) ease-out;
+      }
+
+      &[data-iui-active='true'] {
+        --_iui-button-active-stripe-color: linear-gradient(
+          0deg,
+          var(--iui-color-background-accent-muted) 25%,
+          var(--iui-color-border-accent) 25%,
+          var(--iui-color-border-accent) 75%,
+          var(--iui-color-background-accent-muted) 75%
+        );
+      }
```

Comment:
I really like the visual changes for moving the stripe over for active tabs and removing the stripe and matching the color for the open tabs. It looks good and is a nice improvement.